sql-insert/insert-actor.sql

```diff
@@ -0,0 +1,5 @@
+insert into "actors" ("firstName", "lastName")
+  values ('Sharon', 'Tieu');
+
+select *
+  from "actors";
```

Comment:
Instead of doing a second `select` query you should just be using a `returning` keyword with your `insert` statement.
```SQL
insert into "actors" ("firstName", "lastName")
  values ('Sharon', 'Tieu')
returning *;
```